src/Attestation/Attestation/Properties/AssemblyInfo.cs

```diff
@@ -29,8 +29,8 @@
 [assembly: CLSCompliant(false)]
 
 [assembly: Guid("2994548F-69B9-4DC2-8D19-52CC0C0C85BC")]
-[assembly: AssemblyVersion("0.1.2")]
-[assembly: AssemblyFileVersion("0.1.2")]
+[assembly: AssemblyVersion("0.1.3")]
+[assembly: AssemblyFileVersion("0.1.3")]
```

Comment:
You don't need to update the version, we will update it for you.

Reply:
change the version back to 0.1.2